occurrences/models.py

```diff
@@ -62,6 +70,13 @@ def is_editable_by_user(self, user):
             return user.person.organisations.filter(id=self.organisation.id).exists()
         return True
 
+    def get_end_time_from_occurrences(self):
+        # Return the latest time that teacher can enrol to the event
+        if not self.occurrences.exists():
+            raise ValueError("Palvelutarjotin event has no occurrence")
+        last_occurrence = self.occurrences.order_by("-start_time").first()
```

Comment:
`order_by("start_time").last()` would have been too simple? 😉

Reply:
Yah would have been `latest("start_time")` for maximum simplicity 😆

Reply:
Crap, that is even better 😄 

But actually, now the whole thing would be better like
```
try:
    last_occurrence = self.occurrences.latest("start_time")
except Occurrence.DoesNotExist:
    raise ValueError("Palvelutarjotin event has no occurrence")
```
to avoid one unnecessary DB request and a theoretical race situation.

Reply:
You are the best 🤘 Fixed!